types/file.go

```diff
@@ -8,8 +8,10 @@ var FileLockDefaultTimeout = 24 * time.Hour
 
 type DiskStat struct {
 	DiskID           string
+	Name             string
 	Path             string
 	Type             string
+	Driver           string
```

Comment:
What does `Driver` mean here?

Reply:
Disk driver used by block device. It indicated the disk is managed by aio, nvme or virtio.

Reply:
I see. Maybe we should use `type DiskDriver string` and define those drivers beforehand?

Reply:
Sure!